### PR TITLE
Ooops! Forgot fix the backpack!

### PR DESCRIPTION
**Pull request by pkendall64**
Who reviewed this guys code 😆 

When I made the change for the Logging/Backpack serial port to be a stream I forgot to fix it in the backpack device!

### CODE REVIEW

File: src/lib/Backpack/devBackpack.cpp [hunk 1]
```diff
@@ -36,7 +35,8 @@ void startPassthrough()
 
     // get ready for passthrough
     CRSF::Port.begin(460800, SERIAL_8N1, GPIO_PIN_RCSIGNAL_RX, GPIO_PIN_RCSIGNAL_TX);
-    LoggingBackpack.begin(460800, SERIAL_8N1, GPIO_PIN_DEBUG_RX, GPIO_PIN_DEBUG_TX);
+    HardwareSerial &backpack = *((HardwareSerial*)LoggingBackpack);
+    backpack.begin(460800, SERIAL_8N1, GPIO_PIN_DEBUG_RX, GPIO_PIN_DEBUG_TX);
```
Comment (pkendall64) [inline src/lib/Backpack/devBackpack.cpp:39]: @CapnBry got any better suggestions for this?

**Comment by jurgelenas** [comment]
@pkendall64 what about backpack wifi char fixes?

**Comment by pkendall64** [comment]
> @pkendall64 what about backpack wifi char fixes?

Theres a PR for that in the backpack repo 👍

**Review by AlessandroAU — APPROVED** [review]

File: src/lib/Backpack/devBackpack.cpp (hunk 1 above)
Comment (CapnBry) [reply src/lib/Backpack/devBackpack.cpp:39]: 😬 Well, the LoggingBackpack already runs at `#define BACKPACK_LOGGING_BAUD 460800` so you'd just replace the line with:
```
#if BACKPACK_LOGGING_BAUD != 460800
#error "Passthrough backpack requires BACKPACK_LOGGING_BAUD==460800
#endif
```

All the code after that would still operate on `LoggingBackpack->`.

**Review by CapnBry — APPROVED** [review]
Still works on DupleTX! Do we want to change this too since we have a fixed baud to make it obvious?
```
     // get ready for passthrough
-    CRSF::Port.begin(460800 ...
+    CRSF::Port.begin(BACKPACK_LOGGING_BAUD ...
```

**Comment by pkendall64** [comment]
> Still works on DupleTX! Do we want to change this too since we have a fixed baud to make it obvious?
> 
> ```
>      // get ready for passthrough
> -    CRSF::Port.begin(460800 ...
> +    CRSF::Port.begin(BACKPACK_LOGGING_BAUD ...
> ```

Forgot that one. Will do then I'll merge.

**Pull request merged**